packages/flx/lib/signing.dart

```diff
@@ -165,8 +165,8 @@ ECSignature _asn1ParseSignature(Uint8List signature) {
 }
 
 PublicKey _publicKeyFromPrivateKey(ECPrivateKey privateKey) {
-  ECPoint Q = privateKey.parameters.G * privateKey.d;
-  return new ECPublicKey(Q, privateKey.parameters);
+  ECPoint q = privateKey.parameters.G * privateKey.d;
```

Comment:
@pq was working on making that lint not fire for single-character variables of any case, and the `flutter analyze` command has a hack to hide this specific warning, FWIW.